assets/src/components/kubernetes/Navigation.tsx

```diff
@@ -114,7 +114,7 @@ export default function Navigation() {
           flexShrink: 1,
           height: '100%',
           width: '100%',
-          overflow: 'hidden',
+          minWidth: 0,
         }}
```

Comment:
@jsladerman This solves issue with overflows getting clipped. `overflow: hidden` was used there to ensure that tab scroll will work correctly but we can set `minWidth: 0` instead (`display: flex` sets automatic min size).

Reply:
Oh interesting- nice job